scripts/prettier/prettier-helpers.js

```diff
@@ -10,6 +10,21 @@ const prettierBin = getPrettierBinary();
 const prettierRulesConfig = path.join(repoRoot, 'prettier.config.js');
 const prettierIgnorePath = path.join(repoRoot, '.prettierignore');
 
+const prettierSupportedFileExtensions = [
```

Comment:
we have been executing prettier agains everything which doesn't make any sense. this properly filters to run only on supported files - helping us drastically simplifying `.prettierignore` and using it the proper way